### PR TITLE
New York Times article page feauture

### PR DESCRIPTION
**Pull request by enelesmai**
- Added the nav bar 
- Added main article sections
- Added aside section
- Added inner articles
- Added embedded video
- Added extras section
-Added footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+                <div class="article_date_and_social_icons">
+                    <span class="date">March 17, 2014</span>
+                    <i class="far fa-comments icon_position"></i>
+                    <i class="far fa-bookmark icon_position"></i>
+                    <i class="fas fa-share icon_position"></i>
+                    <i class="fas fa-envelope icon_position"></i>
+                    <i class="fab fa-twitter icon_position"></i>
+                    <i class="fab fa-facebook icon_position"></i>
+                </div>
+                <div style="clear: both;"></div>
```
Comment (mosesogwo) [inline index.html]: remove inline styles

File: index.html [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+                        <span class="date_new">Jan. 19</span>
+                    </div>
+                    <div class="extra_new_a">
+                        <img class="img_new" src="assets/03_Vera_Rubin_gets_telescope.jpg " alt="id_2">
+                        <span class="title_new">Vera Rubin Gets a Telescope of Her Own</span>
+                        <br>
+                        <span class="date_new">Jan. 11</span>
+                    </div>
+                </div>
+                <div style="clear:both"></div>
```
Comment (mosesogwo) [inline index.html]: Remove inline style

File: index.html [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+                    </div>
+                    <div class="extra_new_a">
+                        <img class="img_new" src="assets/06_rocket_launches.jpg" alt="id_5">
+                        <span class="title_new">Rocket Launches, Trips to Mars and More 2020 Space and Astronomy
+                            Events</span>
+                        <br>
+                        <span class="date_new">Jan. 1</span>
+                    </div>
+                </div>
+                <div style="clear:both"></div>
```
Comment (mosesogwo) [inline index.html]: Remove inline style

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Great job team :+1:. A few changes and you can :airplane:**

1. The content of your page below the featured image (the article body) is not centered on the page. Check the positioning on the original page and adjust yours to match. It should be centered.

2. In the **More in Space and Astronomy section**, each article thumbnail with the title should be a clickable link.

3. Adjust the alignment of the article list in the **Most Popular section** so that the left margin is well aligned.

**Yours:**
image

**Original:**
image

3. There is a fixed footer on the original page. Please include it on your own page.
image

4. Remove inline styles from your html markup. It is bad practice.

5. You should remove the Acknowledgements section in your README if there is no one you are acknowledging for this project.

**Kindly make these adjustments and submit another code review request.**

**Cheers:smile:**

**Status: Changes requested**

**Review by Forison — CHANGES_REQUESTED** [review]
## Status Require Changes🛑
Hi team🙌,
You have done a good job so far. You have also implemented some of the changes raised in the previous review. Let address these issues so we can get this project approved
* [x] Files not related to the project should not be committed and added to .gitignore (e.g. .DS_Store) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [x]  Some parts of the page is missing, let us add them
Screenshot from 2020-01-23 18-52-28
* [x] Let us center our main page more than it is now.
## That is it kindly make these changes submit the project for review
### Happy coding💻.
### Addo Forison

**Comment by enelesmai** [comment]
Main page centered and the index added

**Review by addod19 — APPROVED** [review]
## Status: Approved :100: 

Hi TEAM, congratulations on implementing the changes requested by previous reviewers. Your project looks great :+1: 

You can now merge to master and submit project completion form :smile: 
Happy coding :computer: 

Daniel Larbi Addo

**Pull request merged**